### DISCUSSION TITLE
Error: Wrong magic number for PMTiles archive at http://localhost:4200/

### DISCUSSION THREAD

**Discussion by CATALYST1109** [protomaps/PMTiles#211] [Q&A] [answered]
I tried creating my own PMtiles using OSM data thinking I might able to get more local markers than the one from protmaps small map download. I used osm.pbf file and converted it to 5 different geojson files using ogr2ogr. The layers being points,lines,multilinestrings,multipolygons and other-relations. I then used tippecanoe to merge all 5 layers into a single Pmtile using the steps given [here](https://protomaps.com/docs/pmtiles#with-your-own-data). However, when I try using it, I get the error from the title from maplibre gl js when I add the source.

Listing the commands and versions below. Please let me know if you can see the issue somewhere. The osm file is a geofabrik download with correct MD5 hashsum. Thanks!

>ogr2ogr -progress  points-filename-2023-06-28.json filename-2023-06-28.osm.pbf points
>ogr2ogr -progress  lines-filename-2023-06-28.json filename-2023-06-28.osm.pbf lines
>ogr2ogr -progress  multilinestrings-filename-2023-06-28.json filename-2023-06-28.osm.pbf multilinestrings
>ogr2ogr -progress  multipolygons-filename-2023-06-28.json filename-2023-06-28.osm.pbf multipolygons
>ogr2ogr -progress  other_relations-filename-2023-06-28.json filename-2023-06-28.osm.pbf other_relations

>tippecanoe -zg --projection=EPSG:4326 --drop-densest-as-needed -o filename-2023-06-28.pmtiles -l *.json

>tippecanoe --version
tippecanoe v1.35.0

>ogr2ogr --version
GDAL 3.6.4, released 2023/04/17

**Comment by bdon** [comment, marked as answer]
You need to use tippecanoe v2.17 or later which supports pmtiles output. what you're creating in the above is a sqlite (mbtiles) database.

https://github.com/felt/tippecanoe

**Reply by CATALYST1109** [reply]
Thanks!
